ftp/src/test/scala/akka/stream/alpakka/ftp/CommonFtpStageSpec.scala

```diff
@@ -97,4 +99,47 @@ trait CommonFtpSourceSpec extends BaseSpec {
     }
   }
 
+  "FtpIOSink" should {
```

Comment:
I haven't done yet either. It seems that you can control the permissions set with [chmod](http://stackoverflow.com/questions/13150122/set-directory-permissions-chmod-with-jsch).